### ISSUE TITLE
Dumping Segmentation Masks 1.1. takes way too long

### ISSUE THREAD

**Issue by arasharchor** [openvinotoolkit/datumaro#53]
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)

### Expected Behaviour
I have 12 4K images which I have annotated with polygons. When I try to download the annotation with the segmentation mask, it takes a long time. I am still waiting from more 1 hours. The website console seems normal and no errors are issued. 

### Current Behaviour
To my understanding, changing the points to a mask shall be very fast and should not take a long time. 
### Steps to Reproduce (for bugs)

1. upload 12 4K images. 
1. upload annotations for about 3000 polygons.
1. then try to dump the annotations with segmentation mask.

### Context
The images are aerial images and they contain very small to big polygons. 

### Your Environment

- Operating System Linux: Ubuntu 16.04
- Cvat version v1.2.0-alpha

**Comment by zhiltsov-max** [comment]
Hi, could you check server load during exporting? Is it possible that there is high RAM consumption and swapping happens? You can check it, for instance, with `docker exec -it cvat htop`.

**Comment by arasharchor** [comment]
@zhiltsov-max  @nmanovic 

I checked and found out dumping annotations for the Segmentation Mask utilizes a lot of RAM while monitoring HTOP. That was the reason that it was taking too long and crashed after that. 

For instance, I wanted to dump an annotation for an image with the size of 5616x3744px containing polygon annotations.

About **60GB** of RAM was used until the download file got ready. Unfortunately, my laptop crashed due to no having enough RAM, so I resorted a server to be able to dump the annotations. 

To my understanding converting annotation points to an image format shall not use a lot of RAM memory. So I am still wondering what the reason might be.
